internal/status/status.go

```diff
@@ -30,8 +30,9 @@ package status
 import (
 	"errors"
 	"fmt"
+	"google.golang.org/protobuf/proto"
```

Comment:
Please add a newline to separate standard library imports from the rest of the imports.